.github/workflows/superlinter.yml

```
@@ -1,8 +1,7 @@
 ---
 name: Super-Linter
 
-# Run this workflow every time a new commit pushed to your repository
-on: push
+on: [push, pull_request]
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)